ocs_ci/helpers/performance_lib.py

```diff
@@ -748,3 +756,93 @@ def wait_for_resource_bulk_status(
     err_msg = f"{resource.upper()} failed reaching {status} on time"
     logger.error(err_msg)
     raise Exception(err_msg)
+
+
+def pod_csi_time(interface, pv_name, start_time, namespace=constants.OPENSHIFT_STORAGE_NAMESPACE):
+    """
+    Get the starting/ending creation time of a PVC based on provisioner logs
```

Comment:
The comments still says that the function calculates the creation time of PVC? Can you check this again?